
%%%%% Stuff to add:
% plant bio graduate advisor
% botanical conservatory piece highlighting Lulu
%mentoring initiative beta testing
% UC Davis Holistic U Orientation
% review nature plants, elife (x2)

% invited (declined) seminars
% extending knowledge: reporters, master gardener, etc. radiolab
% check GSOC, undergrad mentoring thing for Summer Blanco, add lab training and anti-racist reading

% undergrad research award rankings EVE
% GSMI match
% DEI reading group?
% QEs -- count `em

%%%NOTES TO ADD LATER
%* summer reading group on Coop's NOTES
% appointment and promotion Hernandez
% FOREVER advisory board
% spanish translation for Evolution 2021
% GMI mentoring pilot and feedback

\documentclass[letterpaper,10pt]{article}

\usepackage{hyperref}
\usepackage{geometry}
\usepackage{etaremune}
\usepackage{hanging}
%\usepackage{eurofont}
\usepackage{verbatim}

% Comment the following lines to use the default Computer Modern font
% instead of the Palatino font provided by the mathpazo package.
% Remove the 'osf' bit if you don't like the old style figures.
\usepackage[T1]{fontenc}
\usepackage[sc,osf]{mathpazo}
%\usepackage{setspace}
% Set your name here
\def\name{Jeffrey Ross-Ibarra}

% Replace this with a link to your CV if you like, or set it empty
% (as in \def\footerlink{}) to remove the link in the footer:
\def\footerlink{https://github.com/rossibarra/CV/blob/master/CV.pdf?raw=true}

%Add in-line comments
\newcommand{\ignore}[1]{}

% The following metadata will show up in the PDF properties
\hypersetup{
  colorlinks = true,
  urlcolor = blue,
  pdfauthor = {\name},
  pdfkeywords = {population genetics, maize, plant evolution},
  pdftitle = {\name: Curriculum Vitae},
  pdfsubject = {Curriculum Vitae},
  pdfpagemode = UseNone
}

\geometry{
  body={6.5in, 9in},
  left=1.0in,
  top=1.0in
}

% Customize page headers
\pagestyle{myheadings}
\markright{\name}
\thispagestyle{empty}

% Custom section fonts
\usepackage{sectsty}
\sectionfont{\rmfamily\mdseries\Large}
\subsectionfont{\rmfamily\mdseries\itshape\large}

% Other possible font commands include:
%\ttfamily for teletype,
% \sffamily for sans serif,
% \bfseries for bold,
% \scshape for small caps,
% \normalsize, \large, \Large, \LARGE sizes.

% Don't indent paragraphs.
\setlength\parindent{0em}

% Make lists without bullets
\renewenvironment{itemize}{
  \begin{list}{}{
    \setlength{\leftmargin}{1.5em}
  }
}{
  \end{list}
}

\begin{document}

% Place name at left
{\huge \name}

% Alternatively, print name centered and bold:
%\centerline{\huge \bf \name}

\vspace{0.25in}

%ADDRESS
\begin{minipage}{0.55\linewidth}
  \href{http://eve.ucdavis.edu}{Department of Evolution and Ecology}\\
  \href{http://cpb.ucdavis.edu/}{Center for Population Biology}\\
  \href{http://www.genomecenter.ucdavis.edu/}{Genome Center}\\
  \href{http://www.ucdavis.edu/}{University of California Davis} \\
%  1 Shields Ave.\\
%  Davis, CA 95616
\end{minipage}
\begin{minipage}{0.35\linewidth}
  \begin{tabular}{ll}
    Phone: & (530) 752-4565 \\
    Email: & \href{mailto:rossibarra@ucdavis.edu}{rossibarra@ucdavis.edu} \\
    Web: & \href{http://rilab.ucdavis.edu/}{rilab.ucdavis.edu}, \href{http://www.twitter.com/jrossibarra/}{@jrossibarra} \\
  \end{tabular}
\end{minipage}

%EDUCATION
\section*{Education}
\begin{itemize}
\setlength\itemsep{0ex}
 \item PhD Genetics, University of Georgia 2006
  \item MS Botany, University of California Riverside 2000
 \item BA Botany, University of California Riverside 1998
\end{itemize}

%EMPLOYMENT
\section*{Academic Employment}
\begin{itemize}
\setlength\itemsep{0ex}
\item Professor, Dept. Evolution and Ecology, University of California Davis \hfill 2019-present
\item Scientific Director, UC Davis High Performance Computing Core Facility \hfill2024-present
\item Chair Professor, College of Plant Sciences, Huazhong Agricultural University \hfill2024
\item Paternity leave \hfill2017
\item Professor, Dept. Plant Sciences, University of California Davis \hfill2016-2019
\item Associate Professor, Dept. Plant Sciences, University of California Davis \hfill2012-2016
\item Assistant Professor, Dept. Plant Sciences, University of California Davis \hfill2009-2012
\item Postdoctoral Researcher, University of California Irvine \hfill2006-2008
\item Profesor de Asignatura, Universidad Nacional Aut\'{o}noma de M\'{e}xico \hfill2001
%\item Profesor de Ingl\'es, Boston Language Institute, Ciudad de M\'{e}xico 2000-2001
%\item Field Botanist, University of California Riverside Herbarium 1998
\end{itemize}

%AWARDS
\section*{Selected Fellowships and Awards}
\setlength\itemsep{0ex}
\begin {itemize}
\setlength\itemsep{0ex}
\item NAS Prize in Food and Agriculture \hfill 2024
\item College of Biological Sciences Award for Research Excellence \hfill 2024
\item Fellow, American Association for the Advancement of Science \hfill 2020
%\item Corn Pun Trophy, Genetics Society of America, 2017
\item Stadler Mid-Career Excellence in Maize Genetics Award \hfill 2016
\item Faculty Development Award in recognition of university service \hfill 2015
\item DuPont Young Professor Award \hfill 2012
%\item Academic Senate Travel Award, UC Davis 2010
\item Presidential Early Career Award for Scientists and Engineers \hfill 2009
%\item Dean's Award for Postdoctoral Excellence, UC Irvine 2008
%\item Dissertation Completion Fellowship, University of Georgia 2005-2006
%\item Sigma Xi Grants in Aid of Research 2004
%\item NIH Training Grant, predoctoral research assistantship 2003-2005
%\item University-wide Fellowship, University of Georgia 2001-2003
%\item University of California Germplasm Research Center Grant 2000
%\item Bedding Plants Society Int, John Rathemore Memorial Scholarship 1999-2000
%\item Chancellor's Distinguished Fellowship, UC Riverside 1998-2000
%\item National Science Foundation Research Experience for Undergraduates 1997
%\item Student of the Year, College of Natural and Agricultural Sciences, UC Riverside 1998
%\item Myron Winslow Scholarship 1997-1998
%\item University of California Regents Honorarium 1995-1998
%\item Center for Latin American and Caribbean Studies, Travel Grant 2003
%\item University of California MEXUS Travel Grant 1999
\end{itemize}

%\begin{comment}
%INSTRUCTION & Advising
\section*{Instruction and Advising}
\begin{itemize}
\setlength\itemsep{0ex}
\item Current (total) advisees: 4 (30) postdoc, 6 (13) graduate, 1 (40) undergraduate 

%Thesis Current:
%Mackay Student
%Paul Donat

%Thesis Former:
%Mitchell Feldman (phd, Knapp)
%Jane Park (phd, Whitehead)
%Tianjing (phd, hua chen)
%zane (phd Judy jernstedt)
%chad jorgensen (phd, dvorak)
%Quetzal Orozco (phd, steve brush)
%David O'Donnell (phd, Alan Bennet)
%Gitanshu Munjal (phd, Charlie Brummer)
%Jason Corwin (phd Kliebenstein)
% Michelle Curtis (ms, St. Clair)
\item Graduate Population Genomics Workshop (HZAU, UCD) \hfill 2024-present
\item Hablemos de Evoluci\'on y Ecolog\'ia (First year seminar) \hfill 2025
\item Graduate Advisor, Plant Biology Graduate Group \hfill 2021-present
\item Faculty trainer, Evolution and Ecology Scholars program \hfill 2023-present
\item NSF REU (EEREC) faculty advisor \hfill 2022-present
\item Plant Biology Graduate Core (PLB200A) \hfill 2018-2022
\item Graduate advisor, Ecology Graduate Group \hfill 2018-2021
%\item Instructor, Frontiers and Techniques in Plant Science, CSHL, July 2015
\item Ecological Genomics (ECL 243) \hfill 2014-present
\item Genetics (UC Davis, BIS 101, undergraduate) \hfill 2013-2022
%\item Guest Instructor, UC Davis Plant Breeding Academy, 2012
\item Faculty advisor, US-Mexico graduate student exchange program \hfill 2011-2015
\item Population and Quantitative Genetics (GGG 201D, graduate) \hfill2010-2013 %(5 units, $\frac{1}{2}$ teaching)
%	\subitem Winter 2010, 17 students, instructor rating 4.2, course rating 3.7
%	\subitem Winter 2011, 23 students, instructor rating 4.1, course rating 3.7
\item Plant Genetics (PLS 152, undergraduate) \hfill 2010-2011 %(4 units, $\frac{1}{2}$ teaching)
%	\subitem Fall 2010, 31 students, instructor rating 4.2, course rating 4.0
%\item Guest lecturer, Ethnobotany, Winter 2011
%\item Guest lecturer, Plant Biology Core Course, guest lecturer, Fall 2010 and Fall 2015
%		\subitem Fall 2010 (2 lectures), 9 students, instructor rating 4.4, course rating 3.8
%\item Seminar in plant breeding and biodiversity, 2009%, GGG 292, UC Davis 2009 (2 units)
%		\subitem Fall 2009, 10 graduate students, instructor rating 4.5, course rating 4.5
%\item Guest lecturer, College Success Institute, UCD Academic Preparation Programs, 2009
%\item Open-source Perl, C++, Bash software distribution, 2007-present
%\item Volunteer instructor, NSF Science Behind Our Food Program, 2005
%\item Graduate Mentor, Society for the Study of Evolution Diversity Program 2004
%\item Invited guest lecturer, Genetics, U. Georgia 2003-2005
%\item Graduate Student Teaching Intern for Evolution, U. Georgia 2004
\item Biolog\'{i}a de Plantas I (undergraduate), UNAM \hfill 2001
%\item Teaching Assistant (Genetics, Evolution, Ethnobotany), 1999-2004
%\item Teaching Assistant for Ethnobotany, UC Riverside 1999
%\item Teaching Assistant for Genetics, U. Georgia 2003
\end{itemize}
%\end{comment}

%ADVISING
%\section*{Current Advising}
%\begin{itemize}

%\item PhD Dissertation Committee: Hasan Alhaddad (2013, Genetics), Jason Corwin (Plant Biology), Chad Jorgensen (2017, Hort \& Agronomy), Quetzal Orozco (2016, Anthropology), Jane Park (Genetics), Mengyuan Xiao (Genetics), Erin Calfee (PopBio), Mitchell Feldman (Hort Agron), Kristin Lee (popbio), Erin Calfee (popbio), Kehan Zhao (plant bio), Kersten Schulz (cologne), Kyle Lewald (IGG),
%\item QE Exam Committee: Jeremy Berg (PopBio), Yoong Wearn Lim (Genetics), Stella Hartono (Genetics), Alisa Sedghifar (Pop Bio), Libby Karn (Plant Biology), Gavin Rice (Genetics), 1MS student (Genetics), Colin Contino (PopBio 2015), Perot Saelao (Genetics 2015), Kristin Lee (PopBio 2015), Daniela Rodriguez-Zaccaro (plant bio 2018), Vince Buffalo (2016), Susan Moenga (Genetics 2017), Anita To (2018, Genetics), Syvan Yair (2018, Popbio), Oscar Hinojosa (2018, PlantBio), Jane Park (Genetics, 2017), Lauren Hennelly (2019, Ecology), Cody Bekkereing (2020 plant bio), nathan meier (2019, plant bio),  Kehan Zhao (2021, plant bio), Peter Johnson (IGG 2021), Kirstin Hein (Colorad State 2021),  tianjing zhao (igg, 2021), maddie armstrong (2021, popbio), brooke benson (2021, popbio)
%\item MS Thesis Committee: Joanne Heraty (International Ag. & Development), Michelle Curtis (Genetics)
%\item Advisees:
%	\begin {itemize}
%		\item postdoctoral
%			\begin {itemize}
%				\item Joost van Heerwaarden -> research scientist, Wageningen
%				\item Sofiane Mezmouk -> KWS research scientist
%				\item Tanja Pyh\"aj\"arvi ->professor Helsinki
%				\item Matthew Hufford -> professor Iowa  State
%				\item Shohei Takuno -> professor Sokendai
%				\item Suja George ??
%				\item Sayuri Tsukahara ??
%				\item Kate Crosby -> senior scientist California Department of Fish and Wildlife
%       \item Tim Beissinger -> Professor Goettingen
%				\item Simon Renny-Byfield -> research scientist, Corteva
%				\item Jinliang Yang -> professor U Nebraska
%				\item Luis Avila Bolivar -> research scientist ??
%				\item Emily Josephs -> Profesor Michigan State
%				\item Josh Hough -> deceased
%				\item Wenbin Mei -> research scientist, Inari Ag.
%				\item Dan Gates -> bioinformatiican Checkerspot
%				\item Cathy Rushworth -> professor, Utah State
%       \item Anne Lorant -> bioinformatician, Laboratoire de Biologie Moleculaire et Cellulaire du Cancer
%       \item Sarah Turner -> research scientist, Bayer
%       \item Silas Tittes -> Bioinformatician fulcrum genomics
%       \item Markus Stetter -> Professor, Cologne
%       \item Li Wang -> professor Sehnzhen Ag  Un
%        \item Mitra Menon -> colossal
%     \item Edwin Solares -> UCR
%      \item Ning Yang ->professor HZAU
%        \item Robert Horvath (postdoc)
%   \item beibei liu
%   \item sam snodgrass
%   \item ayelet salman-minkov
%			\end {itemize}
%   \item visiting students
%     \begin{itemize}
%       \item Leo Zeitler
%       Idalia Rojas
%      \end{itemize}
%		\item visiting postdoc
%			\begin {itemize}
%				\item Ginnie Morrison
%       \item Rute Fonseca
%			\end {itemize}
%    \item visiting scientist
%			\begin {itemize}
%				\item Peter Tiffin
%        \item Norm Ellstrand
%        \item Peter Morrell
%        \item Eric Fuchs
%			\end {itemize}
%		\item graduate
%			\begin {itemize}
%				\item Dianne Velasco (Genetics)
%				\item Anna O'Brien (Pop Bio)
%       \item Anne Lorant (France)
%       \item Elli Cryan (Plant Bio)
%   \item Alyssa Phillips (Plant Bio)
%				\item Paul Bilinski (Plant Bio)
%				\item Michelle Stitzer (Pop Bio)
%       \item Asher Hudson (Pop Bio)
%       \item Regin Fairbanks (Pop Bio)
%       \item Sarah Odell (Plant Bio)  
%       \item Natasha Dhamrait (PopBio)
%       \item paulo magalang
%      \item cristian stark
%			\end {itemize}
%		\item undergraduate
%			\begin {itemize}
%        \item Defne Sim <- Mitra/JRI
%       \item Vibha (Dhamrait), 
% \item Varsha (JRI), 
% \item Hana Lubin (Regina), 
% \item Garnett Phinney (Cryan), 
% \item (Alyssa 3 students poster ), 
% \item Sowmya (JRI)
%        \item Joanna <-Alssa/JRI
%				\item Lauryn Brown <-Vann
%				\item Nikhil Ghopal <- JRI
%				\item Thomas Kono <-JRI
%				\item Pui Yan Ho <- Hufford
%				\item Michael May <-JRI
%				\item Evan Pellerin <-JRI
%				\item Lauren Sagara <-JRI
%				\item Casper Thommes <-JRI
%				\item Shobhana Rajasenan <-JRI
%				\item Michael Devengenzo <-Hufford
%				\item Tyler Kent <- JRI
%				\item Timothy Yang <-JRI
%				\item Nivaz Brar <-Inagaki
%				\item Ali Khavari <-Stitzer
%				\item Kevin Distor <- Bilinski
%				\item Chris Fiscus <-JRI
%				\item Gitanshu Munjal <-Beissinger
%				\item Sid Badhra-Lobo <-JRI
%				\item Arun Durvasula <-JRI
%        \item Ashley Johnson  - jrossibarra/asher
%           \item Sean McGinty -JRI
%				\item Juvenal Quezada <- Bilinski
%				\item Olivia Jackson <- Obrien
%       \item Kevin Boardman <- Obrien
%       \item Ellen Osborn <- Stitzer
%       \item Brandon Kung <- Stitzer
%       \item Niki Mahmoudzadeh <- Gates
%       \item Scotty Mahan? <- JRI
%       \item Tanmayee Khadilkar <- Stitzer
%        \item Nathaniel Langlois <- JRI
%			\end {itemize}
%		\item graduate student exchange program
%			\begin {itemize}
%				\item Jorge Alberto
%				\item Gabriela Mendoza
%				\item Aurelio Hernandez Bautista
%				\item Cesar Alvarez
%				\item Pablo Andres
%				\item Jose Gutierrez-Lopez
%        \item Eric Gonzalez
%        \item Chio Aguilar
%				\item Aldo Carmona
%				\item Antonio Hernandez
%			\end {itemize}
%		\end {itemize}
%\end{itemize}

%PROFESSIONAL SERVICE
%TO ADD
% professors for the future grant writing Dec 2024
\section*{Service: selected from last 3 years}
\begin{itemize}
\setlength\itemsep{0ex}
\subsection*{University}
\item Scientific Director, High Performance Computing Core Facility \hfill 2024-present
\item START taskforce on Research Computing \hfill 2024-present
\item Member, Advanced Computing Committee, UC Davis Health \hfill 2025-present
\item Member, Computer Support Committee, College of Letters \& Science, \hfill 2024-present
%\item Chair, CBS High Performance Computing Advisory Cmte \hfill 2021
\item Chair and Co-Chair, Ecology \& Evolution Seminar Series \hfill 2020-2022
\item EVE representative, Faculty Executive Cmte, College of Biological Sciences \hfill 2020-2022
\item Advisory cmte, controlled environment facility \hfill 2020-2024
\item Executive Cmte, Plant Biology Graduate Group \hfill 2019-2025
%\item CPB review 2019 %Mike Miller
% \item maybe? hallway renewal stuff eve
%\item Campus-wide High-Performance Computing Task Force \hfill2019
%\item Hatch/AES review,  Plant Sciences (2018), EVE (2019) % Pat J Brown Dept. Plant Sciences 2018, Chuck Langley EVE 2019
%\item Ecology Graduate Group admissions committee \hfill 2018
\item Confidential committee for the Office of the Vice Provost --- Academic Affairs  \hfill2017-2024 % served as lead on one case 2019,  one 2020
\item \ignore{Todesco 2022} appointment (1), \ignore {Mandel 2023, Cornejo 2022, Grace Lee 2024, Hodgins 2023} tenure (4), \ignore{Lowry 2023, Hernandez 2021} full professor (2)  and distinguished professor \ignore{Dawe 2024} letters \hfill 2022-2025
%award nomination sherry (2023, taba (2023, brandon(2024)))
%Cornell, U Georgia, Michigan State, Nebraska State, UC Irvine, U Iowa, U Minnesota, UC San Francisco,
%Hodgins, Mandel, Lowry
%Lipka (Cornell), Wallace (UGA), Lowry (MSU) 2019
  %* tenure letters (Schnable, Alex, JJ, Lowry)
  % (distinguished prof Tiffin letter)
  % full prof  (Comeron,thornton)
  % appointment and promotion Hernandez
% \item Alternate Dept. Representative, UCD Academic Senate, 2016
% \item Genome Center Bioinformatics Core Advisory Board, 2016
%\item College review, NSF limited submission, 2018
%\item Campus Task Force on Bioinformatics% \hfill2017-2018
%\item Campus High Performance Computing Advisory Committee, 2016-2018
%\item Dept. of Plant Sciences GSR Awards Committee, 2017
%\item UC Davis representative, UC-Mexico Initiative committee on the environment, 2014-2016
%junior faculty mentor committee 2017 %Brown
%\item Acting Chair, Dept. of Plant Sciences, 2015 (3 weeks)
%\item Section Chair for Agricultural Plant Biology\hfill2014-2019
% \item Grad School Panel, HBCU summer program, 2018
%\item Plant Sciences executive committee\hfill2014-2019
%\item Faculty advisor, Corteva graduate student symposium in plant science \hfill2012-2021
%\item UCD Hellman Fellowship review committee, 2014
%\item Center for Population Biology award committee, 2014
%\item Plant Biology seminar committee, 2011,2014
%junior faculty mentor committee 2014 %Runcie
%\item Outside appointment letter, UCSD 2014
%\item College of Ag. and Environ. Sciences Visioning Committee, 2013
%\item Admissions committees: Plant Biology (2013), Population Biology (2013-2015), Ecology (2015)
%\item Search committees: Climate Adaptation (PLS)\hfill2017-2019

% search committees: Director Genome Center Sequencing Core (2014), Science Writer (2012), Pop/Quant Geneticist (2012), Bioinformatician (chair, 2013), Director Plant Breeding Center (2013), Dept. Chair of Plant Sciences (Chair, 2016),
%\item Chair, Dept. of Plant Sciences IT committee, 2011-2013
%\item Executive Committee, Genetics Graduate Group, 2009-2012
%\item College committee on strategic planning for plant breeding, 2011
%\item Dept. of Plant Sciences academic planning committee\hfill2010-2016, 2018-2019

\subsection*{Professional}
\item Organizing Committee, Society for Molecular Biology and Evolution Connference \hfill 2024
\item Chair, Genetics Society of America PEQG Conference  \hfill2022
\item Chair, Plant Genomes Online Conference \hfill 2022
\item Board of Directors, Maize Genetics Corporation \hfill2022-present
\item Maize Genetics Advocacy Cmte, \hfill 2022-present
\item Executive Council, Society of Molecular Biology and Evolution \hfill2022-2024
\item Scientific Advisory Board, FOREVER project \hfill2021-present %- https://www.aka.fi/globalassets/1-tutkimusrahoitus/4-ohjelmat-ja-muut-rahoitusmuodot/1-akatemiaohjelmat/rlife/rlife-hanke-esittely-pyhajarvi_jpg_s_hires.pdf
%\item Organizing Committee, PEQG 2020 Conference\hfill2020
%\item Abstract review, SACNAS \hfill 2020
%\item LEAD21 Leadership in Land Grant Institutions, Class 14%\hfill2018-2019 %%DO%%
\item Founder and organizer, Zeavolution webinar series \hfill2019-present
%2020: UCSF appointment letter, Ryan Hernandez
%2021: UCSF promotion letter, Ryan Hernandez
%\item Maize Genetics conference steering committee\hfill2018-2021
\item Maize Genetics Awards Committee\hfill2017,2019-2020,2022
%\item Advisory Board, PeerJ Preprints \hfill 2016-2019
%Peer Community in Evolutionary Biology (2017-present)
% PLoS Genetics (Guest Editor, 2014, 2017)
%\item Global Crop Diversity Trust Crop Relative Genomics workshop, 2012
%\item 2014 External promotion review letter writer, UCSD, UCR %Barreto, Ashworth, Gore (2016
%\item 2014 External tenure writer, U Minn. %Peter
%2013 How to get into grad school talk to undergrads
%Consultation on maize genetics: IDEA connections (2013),
%\item Science Writer search committee, Dept. Plant Sciences, 2012
%\item Guest editor, PLoS Genetics, 2012
%\item Faculty AES review, Dept. of Plant Sciences, 2012
%\item Plant Sciences faculty mentor, 2012
%\item Scientific Advisory Board, AMAIZING Project (INRA), 2011-present %<- LEAVE OFF
%\item Host, Guggenheim Fellow % (Norm Ellstrand)
%\item Evolutionary genetics seminar chair, Genetics Graduate Group, 2010-2012
%\item Center for population biology nomination committee, 2012
%\item Associate editor, American Journal of Botany 2009-2011
%\item NSF-USDA Phenomics workshop, 2011
%\item Grant panel review (last 2 years): USDA NIFA, UC MEXUS
%UCMEXUS  2011
%DOE 2010?
%NIFA 2011
%\item Minnesota Agricultural Experiment Station, Hatch Project external reviewer, 2010
%\item Journal peer review in 2010-11: Nat. Genetics, PLoS Genetics, PNAS, Genetics, MBE, Heredity, AJB, PLoS ONE
%\item Chair, Plant Breeding and Biodiversity, Genetics Graduate Group, UC Davis 2009-2010
%\item Seminar Committee, Plant Biology Graduate Group, UC Davis, 2009, 2010
%\item  Ad-hoc grant review (last 2 years): BARD, NSF, France ANR, %Agence Nationale de la Recherche, MN Ag. Experiment Station
%MN ag july 2010
% BARD Jan 2012
% NWO Nov. 2011
% ANR Jan 2012
%NSF Feb 2012
%\item Guest editor, PLoS Genetics, 2014 % also 2012
%\item External search committee, Dept. Plant Biology, Swedish University of Agricultural Sciences (2016)
\item Editorial Boards:
Genes, Genomes, and Genetics Senior (2017-2024), Associate (2014-2024), Guest (2025-present) Editor, %(AE 2014-2019, SE 2017-2019),
New Phytologist Associate Editor (2021-2022),  %2021-present
%Genetics (Guest Editor) %, 2018),
Peer Review in Evolutionary Bio (2025-present)
bioRxiv affiliate (2021-present)
eLife Reviewing Editor (2021-2024)
%PLoS Genetics Associate Editor (2018-2019),
%PeerJ Senior (2018-2019) and Associate (2013-2021) Editor
%PNAS Guest  Editor  (2018)
% eLife (Guest, 2016)
    % axios reviews (2013-2017)
%\item Grant peer review: NSF \ignore{1/21}
\item Journal peer review:   \ignore{10/24} Molecular Ecology, \ignore{10/23,10/24} Science (2),   \ignore{2/22,5/22,6/22,2/24,7/23,9/23,2/24} eLife (7),  Genetics \ignore{10/22}, \ignore{2/22} Plant Cell,  Trends in Genetics \ignore{9/21}, Trends in Ecology \ignore{9/21}, \ignore{9/2023} Evolution, \ignore{9/23} Evolutionary Applications, PNAS (5) \ignore{7/21,1/22,5/24,6/24,10/24},  Nature Communications \ignore{10/24}, G3 (3) \ignore{12/22,3/22,11/21,9/21}, \ignore{12/23,8/2023} PLoS Genetics (2), \ignore{8/2021,8/22} American J Botany (2), Current Biology \ignore{1/22}
%\ignore{1/21} The Plant Journal,
% Current Biology \ignore{3/21},
%\ignore{3/21} Molecular Ecology, 
%Scientific Reports \ignore{3/2021}, 
%\ignore{2/20} Cell,
%\ignore{ 6/21} Trends in Genetics
% \ignore{1/21} Nature,%
%\ignore{11/17} Agriculture, Ecosystems and Environment 
%\ignore{11/19,2/22} eLife (2)
%\ignore{2/19} Nature Reviews Genetics,
%\ignore{7/19} New Phytologist,
%Molecular Ecology \ignore{3/21,7/19},
%\ignore{7/19} Science
%\ignore{8/19} PLoS Genetics,
%\ignore{4/20,2/22} Plant Cell (2)
%\ignore{7/20,7/19} Nature Communications (2),  
%\item Proposal review: NSF (2017)
%\item Proposal review: NSF (2016,2017 (2)), Swiss National Science Foundation (2016), GWIS (2016), Israeli Science Foundation (2016)
%\ignore{6/17} The Plant Cell
%\ignore{12/17} New Phyt
%\ignore{4/18,7/18,8/18} PNAS
%\ignore{9/17} Plant Journal
%\ignore{9/17} Cell
%\ignore{1/17,6/17} BMC Genomics (2),
%\ignore{1/17} Genome Biology \& Evolution,
%\ignore{12/16} Scientific Reports,
%\ignore{9/13} Peerage of Science
%\ignore{7/16} Genome Research,
%\ignore{1/14} BMC Genomics
%\ignore{9/15,6/16} Trends in Plant Science
%\ignore{1/14,1/15,1/15,11/16,2/17,5/17} Molecular Ecology
%\ignore{10/14} BMC Biology
%\ignore{1/16}  Nature Plant,
%\ignore{7/13,1/16, 2/16,} Genome Research,
%\ignore{6/16,10/16,10/18,1/19,} eLife
%\ignore{8/13,9/15} Current Biology
%\ignore {3/14} American Naturalist,
%\ignore{5/15,6/15,7/15,9/15,7/14,7/14} Molecular Bilogy and Evolution
%\ignore{12/14,6/15} Genome Bilogy and Evolution
%\ignore{7/13} Scientific Reports
%\ignore {3/13} Economic Botany,
%\ignore{4/13} PLoS Biology,
%\ignore{5/15,6/16} PLoS Genetics
%\ignore{4/13,1/15,6/15} PLoS One,
%\ignore{3/13,4/14,4/14,8/14,12/14,} PLoS Genetics
%\ignore {6/13,7/13,8/13,8/13} PNAS
%\ignore {8/13,10/13,7/15,6/16,7/16,12/16,2/17} Nature
%\ignore{6/15,7/15} Nature Plants
%\ignore{9/17,4/18} Nature Communications
%\ignore{7/15,1/21} Genetics Genetics 
%\ignore{6/15} G3
%\ignore {1/14,3/14,4/16} Nature Genetics
\end{itemize}

% invited (declined) seminars
% extending knowledge: reporters, master gardener, etc. radiolab
% check GSOC, undergrad mentoring thing for Summer Blanco, add lab training and anti-racist reading
% undergrad research award rankings EVE

%%%NOTES TO ADD LATER
%* summer reading group on Coop's NOTES
%* tenure letters (Schnable, Alex, JJ, Lowry)
% (distinguished prof Tiffin letter)
% full prof  (Comeron)
% appointment and promotion Hernandez
% FOREVER advisory board

\subsection*{Contributions to Diversity}
\begin{itemize}
  \setlength\itemsep{0ex}
  \item Maize Genetics code of conduct committee \hfill 2023-present %wrote code of conduct action plan, disciplinary meetings, etc.
  \item Society of Molecular Biolgoy, \textit{ad hoc} code of conduct committe \hfill 2024-2025 %discipline/investiation
  \item Faculty representative, Pop Bio graduate student DEI cmte \hfill 2023-2024
  \item Chair, Maize Genetics review of multi-society DEI initiative \hfill 2023
  %\item Developed harrassment investigation/discipline documentation, Maize Genetics Cooperative
  \item Faculty representative, EEB grad preview workshop \hfill 2023
  \item Member, IDEA Cmte, Society for Molecular Biology and Evolution \hfill2022-2024
  %IDEA workshop, demography, mentoring award for council
 % \item Trainee, EEREC REU program, \hfill 2022-2023
 % \item Member, pilot program, graduate student mentor training \hfill 2021
 % \item Spanish translation: Evolution Conf., UC Master Gardeners \hfill 2021
%  \item Mentor, Graduate Student Mentoring Initiative, Cientifico Latino (2) \hfill 2021 %2 students, multiple meetings each
  %\item Advisor, graduate student of color mentoring program (2) \hfill2020-2021 %(2 students, $\sim45$min per week during academic year)  %2 students ~1hr/week total
 %\item EVE Diversity cmte (organized 2 workshops)  \hfill 2020-2021
 % \item Faculty host, HBCU summer research internship program \hfill 2020
\end{itemize}

\subsection*{Outreach}
\begin{itemize}
  \setlength\itemsep{0ex}
  % \item San Francisco Exploratorium, May 2015
\item Talking Biotech podcast \hfill 2025
\item Corteva ''Breeding Bites'' podcast \hfill 2024
\item KQED Documentary ``Your Corn Tortilla Sucks…Science Can Fix It''\hfill 2024
\item Eat This podcast ``A New Story for Maize Domestication''\hfill 2023
\item San Francisco Exploratorium panel on crop domestication, \hfill 2023
\item Good Food podcast ``Maize is life'' \hfill 2022
\item Expert interview for \ignore{2024} National Geographic, \ignore{2023} Science\&Vie, \ignore{2023} Folha de S.Paulo, \newline \ignore{2023} South China Post \hfill 2022-2025
%\item Radiolab podcast ``body count'' \hfill 2020
%\item
%\item Science and Society (public lecture), Woodland 2018
%\item Skype a Scientist: New Jersey, USA; Burgos, Spain \hfill 2018
%\item Skype-a-Scientist K-12 Outreach: Canada, Spain, New Jersey%2017: Korematsu Elementary, CA; Shelburne Middle School, VA; Summit Hill Elementary, IL;
%2020: Connect Charter School, Canada %\hfill2020
%2018 Espiciencia, Spain; Valleyview Middle School, NJ%\hfill2018
\end{itemize}

%CURRENT FUNDING
%\section*{Current Funding}
\begin{itemize}
% \setlength\itemsep{0ex}
\item NSF: ``REUSite: Ecology,Evolution,and Equity in Environmental Change (4EC)'' \$350,000 (PI) \textit{Recommended for Funding} 2024-2027
\item NSF: ``DISES: Coevolutionary dynamics of humans and maize in the Americas'' \$1.6M (PI), 2023-2026
\item Gates Foundation: ``Study and test approaches for identifying alleles associated with environmental adaptation'' \$320,000 (Co-PI), 2022-2026
\item NSF: ``PGRP: Uncovering the role of transposons in maize variation'' \$800,000  (Co-PI), 2019-2025%
%   %\item UC Davis Global Affairs Initiative (PI), 2016-2017
%  \item NSF: ``Harnessing convergence and constraint to predict adaptations to abiotic stress for maize and sorghum'' \$740,000 of \$5.4M total (Co-PI), 2018-2022
%  \item NSF: ``The evolutionary genetics of pollen-pistil incompatibility and reproductive isolation in \textit{Zea mays}'' \$320,000 of \$1M total (Co-PI), 2018-2022
%  \item USDA:
%  ``Mapping the genetic loci controlling highland adaptation in maize'', \$29,000 (PI) 2021-2023
%  \item NSF: ``The genetics of highland adaptation in maize'', \$4.2M (PI), 2016-2022
% % %\item UC-Mexico Initiative: "Maize adaptation to climate in Mexico" (PI), 2015-2016
% % \item UCMEXUS: "Adaptive gene flow from teosinte to highland maize in central Mexico" (Co-PI), 2015-2016
% % \item UC-Mexico Initiative: "Maize adaptation to climate in Mexico", 2015-2016 (PI, \$30,000 total)
% % \item Smithsonian Institution: “Phenotypes, environmental plasticity, and gene expression in teosinte and maize in ancient climates before and during the time of domestication” (Co-PI) 2015-2017
% % \item UCMEXUS: "Adaptive gene flow from teosinte to highland maize in central Mexico", 2015-2016 (Co-PI, \$24,897 total)
% % \item NSF: ``Biology of rare alleles in maize and its wild relatives'' \$3.2M of \$13M total (Co-PI), 2013-2019
% % %\item NSF Plant Genome Research Program: "Biology of Rare Alleles" (Co-PI, \$3.2M to JRI), 2013-2018
% % %\item NSF Plant Genome Research Program: "The genetics of highland adaptation in maize" (recommended for funding; PI, \$4.1M), 2016-2021
% % \item NSF Plant Genome Research Program (Co-PI, \$754,409 to JRI, 2010-2016) "Functional Genomics of Maize Centromeres"
% % \item USDA-NIFA Plant Genome, Genetics, and Breeding: "Scanning the weather: high-throughput discovery of agronomic loci for advanced maize breeding to address climate change" (PI, \$300,000 Sept 2012-Sept. 2014)
% % %\item USDA NIFA Plant Genome, Genetics, and Breeding (PI, \$748,000, Sept. 2009 - Sept. 2014)
% % \item USDA-ARS: ''Collection of the maize wild relative, \emph{Zea luxurians}, in southeast Guatemala for ex situ conservation'' (PI, \$9,000) Nov 2010 - June 2012
% % \item France-Berkeley Fund (PI, \$8,160 Jan 2012-June 2013)
% % %\item USDA ARS Plant Germplasm Collection (PI, \$9,000, Nov. 2010-Dec. 2012)
% % \item USDA-NIFA Plant Genome, Genetics, and Breeding: ''Scanning for yield: high-throughput discovery of candidate agronomic loci for marker-assisted selection in maize'' (PI, \$448,000 Sept 2009 - Sept 2012)
% % \item UC MEXUS: ''Phylogeography and Systematics of Mesoamerican \emph{Diospyros}''  (PI, \$14,825) July 2009 - Jan 2012
% % \item France-Berkeley Fund: "Does domestication affect recombination: a pilot study in maize" (PI, \$8,160 Jan 2012-June 2013)
\end{itemize}

%SEMINARS
\section*{Invited Seminars, previous 3 years}
\begin{itemize}
\setlength\itemsep{0ex}
%\item Society of American Archaeology, April 2023
\item GRC in Ecological \& Evolutonary Genomics (keynote), July 2025
\item UC Davis, May 2025
\item UC Berkeley, May 2025
\item Gomez-Pompa lecture, UC Riverside, May 2025
\item Maize Biology Conference of China (keynote), April 2025
\item Darwin Day lecture, U. Calgary, Feb 2025
\item Huazhong Agricultural University, Aug. 2024
\item Shenzhen Agricultural Genomics Instittue, Aug. 2024
\item Yazhouwan National Laboratory, Aug 2024
\item Sacramento Archeological Society, July 2024
\item Corteva Agriscience, June 2024
\item U. Cologne, Mar. 2024
\item U. Oregon, Feb. 2024
\item U. Helsinki, Jan. 2024
\item MexPopGen International Conference (keynote), Oct. 2023
\item Danforth Center, Aug. 2023
\item Bayer Crop Science, Aug. 2023
\item U. Missouri, Aug. 2023
\item Forest Genetics Conference (keynote), July 2023
\item Google X, July 2023
\item Plant and Animal Genome Conference, Jan 2023
\item Iowa State University, Sep. 2022
%\item Advances in Genome Biology and Technology (keynote), April 2022
%\item IPK Gatersleben, Feb, 2022
%\item UC Davis Plant Biology, Feb. 2022
%\item U. British Columbia, Oct 2021
%\item U. Zurich, April 2021
%\item Gates Foundation (plenary), April 2021
%\item U. Vienna, Mar. 2021
%\item Fresno State U, Jan. 2021
%\item Carnegie Institution for Science, Dec. 2020
%\item Calvin Sperling Memorial Biodiversity Lectureship, CSSA meeting, Nov 2020
%\item NC State, Nov 2020
%\item Stony Brook U, Sept. 2020
%\item U. Oulu, April 2020
%\item NYU Abu Dhabi, Feb. 2020
%\item U. Oregon, Feb 2020
%\item U. Massachusetts, Nov 2019
%\item Chinese Academy of Agricultural Sciences, June 2019
%\item U. of Science and Technology, Beijing, June 2019
%\item International Forum on Crop Science, Wuhan, June 2019
%\item UC Riverside, May 2019
%\item U. Vermont, Apr 2019
%\item U. Washington, Mar 2019
%\item Seed Central (keynote), Davis, Feb 2019
%\item James Brewbaker Lecture on Genetics \& Plant Breeding, U. Hawaii, Dec. 2018
%\item National Science Foundation, Washington DC, Sept 2018
%\item U. Georgia, Athens, Aug 2018
%\item Fisher Biosciences, July 2018
%\item Texcoco pioneer symposium Sept. 2018 (declined)
%\item Corteva Agriscience, Johnston, July 2018
%\item Plenary Speaker, Maize Genetics Conference,  Saint-Malo, France, Mar 2018
% \item University of Paris-Saclay, Gif-sur-Yvette, France, Mar 2018
% \item U. California, Davis, Mar 2018
% \item Plant And Animal Genome Conference (two workshops), San Diego, Jan 2018
% \item Plant And Animal Genome Conference (big data) (declined), Jan 2018
% \item U. Nebraska, Lincoln, Nov 2017
% \item U. Colorado, Boulder, Oct 2017
% \item SMBE structural variation symposium, San Antonio, July 2017
% \item Harris Moran Breeding, Davis, Feb 2017
% \item PAG next-generation sequencing symposium, Jan 2017 (declined)
% \item VI Brazilan Plant Molecular Genetics symposium, Ouro Preto, Brazil, May 2017 (declined)
% \item Davis Science Cafe, Aug 2016
% \item SMBE domestication symposium, Queensland, Australia, July 2016
% \item UC Master Gardeners, May 2016
% \item U. Arizona, Apr. 2016
% \item Jo 6 conferencesGenome Institute, Mar. 2016
% \item U. Southern California, Feb. 2016
% \item LANGEBIO, Irapuato, Mexico, Nov. 2015
% \item U. Toronto, Oct. 2015
% \item Danforth Center, Sept. 2015
% \item SMBE workshop on adaptation and next-gen sequencing, Montpellier, June 2015
% \item Dept. of Ecology and Evol. Bio, UC Irvine, April 2015
% \item Cornell Plant Breeding Symposium, March 2015
% \item LANGEBIO (Irapuato), Sept. 2014
% \item Pioneer Hi-Bred (IA), Sept. 2014
% \item Dept. of Ecology and Evolution, Iowa State U., Sept. 2014
% \item Pioneer Hi-Bred (CA), Aug. 2014
% \item Bioagricultural Sciences and Pest Management, Colorado State U., May 2014
% \item Plant Breeding Genetics and Biotechnology Program, Michigan State (MI), Apr. 2014
% \item National Maize Improvement Center of China, China Agricultural University (Beijing), Mar. 2014
% \item Dept. of Agronomy, University of Guelph, Feb. 2014
% \item Plant and Animal Genome Conference, maize workshop, Jan. 2014
% \item Plant and Animal Genome Conference, symposium on domestication, Jan. 2014
% \item Featured Speaker, Ecological Genomics Symposium, Ecological Genetics Institute (MO), Nov. 2013
% \item Department of Genetics, U. Georgia, Sept. 2013
% \item Plenary Speaker, Canadian Plant Genomics Workshop (Halifax) Aug. 2013
% \item Organizer, Evolutionary Genomics symposium, ASPB (RI) 2013
% \item Biodesign Institute, Arizona State U. 2013
% \item Interdisciplinary Plant Group, U. Missouri 2013
% \item UCD@BGI featured speaker, UC Davis 2013
% \item Plant and Animal Genome Conference, symposium on translational genomics (CA) 2013
% \item Featured Speaker, UC Davis Seed Central 2013
% \item Crop Wild Relative Genomics meeting (CA) 2012
% \item Germplasm Enhancement of Maize, ASTA Conference (IA) 2012
% \item Pioneer Hi-Bred (CA) 2012
% \item Plenary Speaker, Coastwide Salmonid Genomics Conference (CA) 2012
% \item BASF Plant Science (NC) 2012
% \item Pioneer Hi-Bred (IA) 2012
% \item Illinois Corn Breeders School (IL) 2012
% \item Keynote Speaker, North Central Regional Corn Breeding Research Meeting (IL) 2012
% \item Plant and Animal Genome Conference, symposium on ecological genomics (CA) 2012
% \item ASA/CSSA/SSSA Convention, symposium on maize biology (TX) 2011
% \item Dept. of Plant \& Microbial Biology, UC Berkeley 2011
% \item Seminis Vegetable Seeds (CA) 2011
% \item Dept. of Plant Sciences, UC Davis 2011
% \item Center for Population Biology, UC Davis 2011
% \item Dept. of Botany and Plant Sciences, UC Riverside 2011
% \item USDA Agricultural Research Service, Iowa State U. 2010
% \item Microbial and Plant Genomics Institute, U. Minnesota 2010
% \item Society for Molecular Biology and Evolution, Plant Ecological Genomics Symposium (France) 2010
% \item Dept. of Plant Sciences, UC Davis 2009
% \item Instituto de Ecolog\'{i}a, Universidad Nacional Aut\'{o}noma de M\'{e}xico 2008
% \item Harlan II Symposium, UC Davis 2008
% \item Dept. of Biology, UC Riverside 2008
% \item Secretar\'{i}a de Medio Ambiente y Recursos Naturales, GMO Risk Assessment (Mexico) 2008
% \item Dept. of Plant Sciences, UC Davis 2007
% \item Dept. of Biology, York University 2007
% \item Dept. of Botany and Plant Sciences, UC Riverside 2007
% \item Georgia Partnership for Reform in Science and Mathematics (PRISM), U. Georgia 2004
% \item University of Georgia Chapter of Sigma-Xi, U Georgia 2004

\end{itemize}

%PUBS
\newpage
\section*{Publications {\small(lab members bold, $^*$equal contribution, $^\ddagger$undergraduate, $^\S$corresponding, [citations])}}

%$^\dagger$cover article

\subsection*{Preprints}
\begin{itemize}
  \setlength\itemsep{0ex}

  \item Stitzer MC, Seethara AS, Scheben A, Hsu S-K, Schulz AJ, AuBuchon-Elder T,   El-Walid M, Ferebee TH, Hale CO,  La T,  Liu Z-Y,  McMorrow SJ,  Minx P, \textbf{Phillips AR},  Syring M,  Wrightsman T,  Zhai J,  Pasquet R,  McAllister C,  Malcomber S,  Traiperm P,  Layton D,  Zhong J,  Costich DE, Dawe RK,  Fengler K,  Harris C,  Irelan Z,  Llaca V,  Parakkal P,  Zastrow-Hayes G,  Woodhouse MR,  Cannon EKS,  Portwood J,  Andorf CM,  Albert PS, Birchler JA,  Siepel A, \textbf{Ross-Ibarra J}, Romay MC,  Kellogg E,  Buckler ES, Hufford MB.  Extensive genome evolution distinguishes maize within a stable tribe of grasses. \\ doi: 10.1101/2025.01.22.633974  
%CITES:15581864934816424318  

\end{itemize}

%PRINT/PRESS
\subsection*{In press or in print} %don't edit below line on H-index, use script instead
H-Index 21 {\small(1549 citations as of Wed Jan 29 11:08:44 2014)}
\begin{etaremune}
\setlength\itemsep{0ex}

\item \textbf{Fairbanks R}, \textbf{Ross-Ibarra J} (2025). An ancient origin of the naked grains of maize. \textsc{PNAS} \textit{Accepted}%\\ %doi: 10.1101/2024.12.02.626434  
%CITES:0  

\item \textbf{Cryan E}, \textbf{Phinney G}, Seetharam AS, Evans MMS, Kellogg EA, Zhan J, Meyers BC,  Kliebenstein DE, \textbf{Ross-Ibarra J} (2025). Molecular evolution of a reproductive barrier in maize and related species. \textsc{Genetics} \textit{In Press} %doi: 10.1101/2024.12.02.626474  
%CITES:0  

\item \textbf{Li F}, \textbf{Gates DJ}, Buckler ES, Hufford MB, Janzen GM, Reall\'an-\'Alvarez R, Rodr\'igues-Zapata F, Romero Navarro JA, Sawers RJH, \textbf{Snodgrass SJ}, SonderK, Willcox MC, Hearne SJ, \textbf{Ross-Ibarra J}, Runcie DE (2025).  The utility of environmental data from traditional varieties for climate-adaptive maize breeding. \textsc{PLoS Genetics} \textit{In Press} % doi: 10.1101/2024.09.19.613351  
%CITES:11400685212099843186  

\item Engelhorn J, \textbf{Snodgrass SJ}, Kok A, Seetharam AS, Schneider M, Kiwit T, Singh A, Banf M, Khaipo-Burch M, Runcie DE, Sánchez Camargo V, Torres-Rodriguez JV, Sun G, Stam M, Fiorani F, Schnable JC, Bass HW, Hufford MB, Stich B, Frommer WB, \textbf{Ross-Ibarra J}, Hartwig T (2025). Phenotypic variation in maize can be largely explained by genetic variation at transcription factor binding sites. \textsc{Nature Genetics} \textit{In Press}
%CITES:3464702864064592867

\item Andorf CM, \textbf{Ross-Ibarra J}, Seetharam AS, Hufford MB,  Woodhouse MR (2024). A unified VCF data set from nearly 1,500 diverse maize accessions and resources to explore the genomic landscape of maize. \textsc{G3} jkae281
%CITES:10121709138180957198

\item \textbf{O'Brien AM},  Sawers RJH, Gasca-Pineda J, Baxter I, Eguiarte LE, \textbf{Ross-Ibarra J}, Strauss SY (2024).  Teosinte populations exhibit weak local adaptation to their rhizosphere biota despite strong effects of biota source on teosinte fitness and traits. \textsc{Evolution} 78: 1991-2005  
%CITES:16508598121234258701,3304032280356441643

\item Berube B, Ernst E, Cahn J, Roche B, de Santis Alves C, Lynn J, Scheben A, Siepel A, \textbf{Ross-Ibarra J}, Kermicle J, Martienssen RA (2024). Teosinte Pollen Drive guides maize domestication and evolution by RNAi. \textsc{Nature} 633: 380-388 
%CITES:10272593141167204712

\item \textbf{Tittes S}$^\S$, \textbf{Lorant A}, \textbf{McGinty S}$^\ddagger$, Doebley JF,  Holland JBH, S\`{a}nchez-Gonz\`{a}lez JdJ, Seetharam A, Tenaillon M, \textbf{Ross-Ibarra J}$^\S$ (2023). Not so local: the population genetics of convergent adaptation in maize and teosinte. eLife12:RP92405 %doi: 10.1101/2021.09.09.459637
%CITES:18080125661207684272,859611454737340250

\item \textbf{Yang N}$^*$, Wang Y$^*$,  Liu X, Jin M, Vallebueno-Estrada M, Calfee E, Chen L, Dilkes BP, Gui S, Fan X, Harper TK, Kennett DJ, Li W, Lu Y, Luo J, \textbf{Mambakkam S}$^\ddagger$, \textbf{Menon M}, Snodgrass S, Veller C, Wu S, Wu S, Xiao Y, Yang X, Stitzer MCS, Runcie DE, Yan J$^\S$, \textbf{Ross-Ibarra J}$^\S$ (2023). Two teosintes made modern maize. \textsc{Science} 382: eadg8940.
%CITES:7069498466040704503

\item Khaipho-Burch M, Cooper M, Corrsa J, de Leon N, Holland J, Lewis R, McCouch S, Murray SC, Rabbi I, Ronald P, \textbf{Ross-Ibarra J}, Weigel D, Yan J, Buckler ES (2023). Genetic modification can improve crop yields --- but stop overselling it. \textsc{Nature} 621:470-473.
%CITES:16268285809405443178,469567953459816391

\item Flint-Garcia S,  Feldmann MJ, Dempewolf H,  Morrell PL, \textbf{Ross-Ibarra J} (2023). Diamonds in the Not-So-Rough: Wild Relative Diversity Hidden in Crop Genomes \textsc{PLoS biology} 21: e3002235	
%CITES:16677551820371444052

\item Sun S, Wang B, Li C, Xu G, Yang J, Hufford MBH, \textbf{Ross-Ibarra J}, Wang H, Wang L (2023). Unraveling prevalence and effects of deleterious mutations in maize elite lines across decades of modern breeding \textsc{Molecular Biology and Evolution} 40: msad170
%CITES:17930901947040544584

\item \textbf{Phillips AR}$^*$, Seetharam AR$^*$, Albert PS, AuBuchon-Elder T, Birchler JA, Buckler ESB,    Gillespie LJ, Hufford MB,    Llaca V,  Romay MC,  Soreng RJ, Kellogg E, \textbf{Ross-Ibarra J} (2023). A happy accident: a novel turfgrass reference genome. \textsc{G3} 13:jkad073
%CITES:11937161593355680525

\item Hu H, Crow T, Nojoomi S, Schulz, AJ, Hufford MB, Flint-Garcia SF, Sawers RJ, Rell\`{a}n-\`{A}lvarez R, Est\`{e}vez-Palmas JM, \textbf{Ross-Ibarra J}, Runcie DE (2023). Allele-specific expression reveals multiple paths to highland adaptation in maize. \textsc{Mol. Bio. Evol.} 39: msac239
%CITES:7367173473048507714

\item \textbf{Rushworth CA}, Wardlaw AM, \textbf{Ross-Ibarra J}, Brandvain Y (2022). Conflict over fertilization underlies the transient evolution of reinforcement. \textsc{PLoS Biology} 20: e3001814 %doi: 10.1101/2020.11.10.377481v1
%CITES:16401606764419060059

\item Chen L$^*$, Luo J$^*$, Minliang Jin$^*$, \textbf{Yang N$^{*\S}$}, Liu X, Peng Y, Li W, \textbf{Phillips AR}, \textbf{Cameron B}, Bernal J, Rell\'{a}n-\'{A}lvarez R, Saers RJH, Liu Q, Yin Y, Ye X, Yan J, Zhang Q, Zhang X, Wu S, Gui S, Wei W, Wang Y, Luo Y, Jiang C, Deng M, Jin M, Jian L, Yu Y, Zhang M, Yang X, Hufford MB, Fernie AR, Warburton ML, \textbf{Ross-Ibarra J$^\S$}, Yan J$^\S$ (2022). Genome sequencing reveals evidence of adaptive variation in the genus \textit{Zea}. \textsc{Nature Genetics} 54: 1736–1745
%CITES:7191906505151277407,14787094829126316446,1364901072355254494,12822792861799271249

\item Li C, Guan H, Jing X, Li Y, Wang B, Li Y-X , Liu X, Zhang D, Liu C, Xie X, Zhao H, Wang Y, Liu J, Zhang P, Hu G, Li G, Li S, Sun D, Wang X, Shi Y, Song Y, Jiao CZ$^\S$, \textbf{Ross-Ibarra J}$^\S$, Li Y$^\S$, Wang T$^\S$, Wang H$^\S$ (2022). Genomic Insights into Historical Improvement of Heterotic Groups during Modern Hybrid Maize Breeding. \textsc{Nature Plants} 8: 750-763
%CITES:18111976045562432877

\item Guerra-Garcia A, Rojas-Barrera IC, \textbf{Ross-Ibarra J}, Papa R, Pi\~nero D (2022). The genomic signature of wild-to-crop introgression during the domestication of scarlet runner bean (\textit{Phaseolus coccineus L.}). \textsc{Evolution Letters} 6: 295-307 %doi: 10.1101/2021.02.03.429668v1
%CITES:15983575394020585674

\item Barnes AC, Rodr\'iguez-Zapata F,Bl\"{o}cher-Ju\'arez KA, \textbf{Gates DJ}, Kur A,  Wang L, Janzen GM,  Jensen S, Est\'evez-Palmas JM, Crow T, Taylor Crow, Aguilar-Rangel R, Demesa-Arevalo E, Skopelitis T, P\'erez-Lim\'on S, Stuttsa WL, Chiu Y-C, Jackson D, Fiehn O, Runcie D, Buckler ES, \textbf{Ross-Ibarra J}, Hufford M, Sawers RJH, Rell\'an-\'Alvarez R (2022). An adaptive teosinte mexicana introgression modulates phosphatidylcholine levels and is associated with maize flowering time \textsc{PNAS} 119: e2100036119  %doi: 10.1101/2021.01.25.426574
%CITES:2541886074262287878,8924460428753965278

\item \textbf{Horvath R}$^\S$, \textbf{Menon M}, Stitzer M, \textbf{Ross-Ibarra J}$^\S$ (2022). Controlling for Variable Transposition Rate with an Age-Adjusted Site Frequency Spectrum. \textsc{Genome Biology and Evolution}  14: evac016 %doi: 10.1101/2021.08.16.456262
%CITES:14778485542675494915

\item \textbf{Hudson AI}, \textbf{Odell SG}, Dubreuil P, Tixier M-H, Praud S, Runcie DE, \textbf{Ross-Ibarra J} (2022).  Analysis of genotype by environment interactions in a maize mapping population. \textsc{G3} 12: jkac013 % doi: 10.1101/2021.07.21.453280
%CITES:6251633562887283614

\item  Samayoa LF, Olukolu  BA, Yang CJ, Chen Q, Stetter MG, York AM, Sanchez-Gonzalez JJ,  Glaubitz JC, Bradbury PJ,  Romay MC, Sun Q, Yang J, \textbf{Ross-Ibarra J}, Buckler ES, Doebley JF, and Holland JB (2022).   Domestication reshaped the genetic basis of inbreeding depression in a maize landrace compared to its wild relative, teosinte. \textsc{PLoS Genetics} 17: e1009797 %doi:10.1101/2021.09.01.458502v1
%CITES:6632947253215446952

\item Perez-Lim\`{o}n S, Li M, Cintora-Martinez GC, Aguilar-Range MR, Salazar-Vidal MN, Gonz\`{a}lez-Segovia E, Blocher-Ju\`{a}rez K, Guerrero-Zavala A, Barrales-Gamez B, Carcano-Macias J,  Nieto-Sotelo J, Martinez de la Vega O, Simpson J, Hufford MB, \textbf{Ross-Ibarra J}, Flint-Garcia S, Diaz-Garcia L, Rell\`{a}n-\`{A}lvarez R, Sawers RJH (2022). A B73 x Palomero Toluque\~{n}o mapping population reveals local adaptation in in Mexican highland maize. \textsc{G3} 12: jkab447 %doi: 10.1101/2021.09.15.460568
%CITES:4754725764254370698

\item \textbf{Odell SG}, \textbf{Hudson AI}, Praud S, Dubreuil P, Tixier M-H, \textbf{Ross-Ibarra J}, Runcie DE (2022). Modeling allelic diversity of multi-parent mapping populations affects detection of quantitative trait loci. \textsc{G3} 12: jkac011 %doi: 10.1101/2021.07.14.452335
%CITES:4200777730517717106
\vspace{0.5cm}

\item Calfee E$^\S$, \textbf{Gates DJ}, \textbf{Lorant A}, \textbf{Perkins MT}, Coop GM$^\S$, \textbf{Ross-Ibarra J}$^\S$ (2021). Selective sorting of ancestral introgression in maize and teosinte along an elevational cline. \textsc{PLoS Genetics} 17: e1009810
%CITES:6213467718649794463

\item \textbf{Stitzer MC}$^\S$, Anderson SN, Springer NM, \textbf{Ross-Ibarra J} (2021). The Genomic Ecosystem of Transposable Elements in Maize. \textsc{PLoS Genetics} 17: e1009768
%CITES:9872302690859149380

\item Hufford MB, Seetharam AS, Woodhouse MR, Chougule KM, Ou S, Liu J, Ricci WA, Guo T, Olson A, Qiu Y Della Coletta R, \textbf{Tittes S}, \textbf{Hudson AI},  Marand AP, Wei S Lu Z, Wang B, Tello-Ruiz MK, Piri R, Wang N, Kim D, Zeng Y, O'Connor CH, Li X, Gilbert AM, Baggs E, Krasileva KV, Portwood JL, Cannon EKS, Andorf CM, Manchanda N, Snodgrass SJ, Hufnagel DE, Jiang Q, Pedersen S, Syring ML, Kudrna DA, Llaca V, Fengler K, Schmitz RJ, \textbf{Ross-Ibarra J}, Yu J, Gent JI, Hirsch CN, Ware D, Dawe RK (2021). De novo assembly, annotation, and comparative analysis of 26 diverse maize genomes. \textsc{Science} 373:655-662
%CITES:7617576175906322793

\item Song CB, Wang H, Wu, Y, Rees E, \textbf{Gates DJ}, Burch M,  Bradbury PJ, \textbf{Ross-Ibarra J}, Kellogg EA, Hufford MB, Romay MC, Buckler ES (2021).  Constrained non-coding sequence provides insights into regulatory elements and loss of gene expression in maize. \textsc{Genome Research} gr.266528.120 %doi: 10.1101/2020.07.11.192575
%CITES:4378530448644595594,13172941186413574025

\item \textbf{Wang L}, Josephs EB, Lee KM, Roberts LM, Rell\'{a}n-\'{A}lvarez R, \textbf{Ross-Ibarra J}$^\S$, Hufford MB$^\S$ (2021). Molecular parallelism underlies convergent highland adaptation of maize landraces. \textsc{Molecular Biology and Evolution} msab119 %doi: 10.1101/2020.07.31.227629
%CITES:5656251523520837481

\item Muyle A, \textbf{Ross-Ibarra J}, Seymour DK, Gaut BS (2021). Gene body methylation is under selection in \textit{Arabidopsis thaliana}. \textsc{Genetics} 218(2):iyab061 %doi:10.1101/2020.09.04.283333
%CITES:16295130435365990103,2456547071832294344

\item Lozano R, Gazave E, dos Santos JPR, Stetter MG, Valluru R, Bandillo N, Fernandes SB, Brown PJ, Shakoor N, Mockler T, Cooper EA, \textbf{Perkins MT}, Buckler ES, \textbf{Ross-Ibarra J}$^\S$, Gore M$^\S$ (2021). Comparative evolutionary analysis and prediction of deleterious mutation patterns between sorghum and maize. \textsc{Nature Plants} 7: 17-24  %  doi: 10.1101/777623
%CITES:11265103186217917709,1423804503041853700

\item \textbf{Ross-Ibarra J}, Piperno D (2020). Maize moving. Figshare. doi: 10.6084/m9.figshare.12781307.v1 [1] 
\newline

\item Xu G, Lyu J, Li Q, Liu H, Wang D, Zhang M, Springer NM, \textbf{Ross-Ibarra J}, Yang J (2020). Adaptive evolution of DNA methylation reshaped gene regulation in maize \textsc{Nature Communications} 11: 5539
%CITES:987817177097090596,14403816878568058683

\item Chen Q, Samayo LF, Yang CJ, Bradbury PJ, Olukolu BA, Neumeyer MA, Romay, MC, Sun Q, \textbf{Lorant A}, Buckler ES, \textbf{Ross-Ibarra J}, Holland JB, Doebley JF (2020).
The genetic architecture of the maize progenitor, teosinte, and how it was altered during maize domestication \textsc{PLoS Genetics} 16.5:e1008791.
%CITES:7610480527513064046

\item \textbf{Zeitler L}, \textbf{Ross-Ibarra J}$^\S$, \textbf{Stetter MGS}$^\S$ (2020). Selective loss of diversity in doubled-haploid lines from European maize landraces. \textsc{G3} 10: 2497-2506
%CITES:40028831341852099,10343727954691350214

\item Wang B, Lin Z, Li X, Zhao Y, Zhao B, Wu G, Ma X, Wang H, Xie Y, Li Q, Song G, Kong D, Zheng Z, Wei H, Shen R, Chen C, Meng Z, Wang T, Li X, Chen Y, Lai J, Hufford MB, \textbf{Ross-Ibarra J}, He H, Wang H (2020). Genome-wide selection and genetic improvement during modern maize breeding. \textsc{Nature Genetics} 52: 565-571
%CITES:3748829117619888894

\item Torres R$^*$, \textbf{Stetter MG}$^*$, Hernandez R$^\S$, \textbf{Ross-Ibarra J}$^\S$ (2020). The temporal dynamics of background selection in non-equilibrium populations. \textsc{Genetics} 214: 1019-1030
%CITES:445881558823431528

\item \textbf{Turner-Hissong SD}$^\S$, Mabrey ME, Beissinger TM, \textbf{Ross-Ibarra J}, Pires JC (2020). Evolutionary insights into plant breeding. \textsc{Current Opinion in Plant Biology} 54: 93-100
%CITES:1066608088401708066

\item  Anderson SN, \textbf{Stitzer MC},  Zhou P, \textbf{Ross-Ibarra J}, Hirsch CD, Springer NM (2019) Dynamic patterns of transcript abundance of transposable element families in maize. \textsc{G3} 9: 3673-3682
%CITES:13372045702551332348

\item  Anderson SN$^*$, \textbf{Stitzer MC}$^*$,  Brohammer A$^*$, Zhou P, Noshay JM,  O'Connor CH, Hirsch CD, \textbf{Ross-Ibarra J}, Hirsch CN, Springer NM (2019). Transposable elements contribute to dynamic genome content in maize. \textsc{The Plant Journal} 100: 1052-1065
%CITES:8452519309966999214

\item Wei X,  Anderson SN,  Wang X,  Yang L, Crisp PA,  Li Q,  Noshay J, Albert PS, Birchler JA,  \textbf{Bilinski P}, \textbf{Stitzer MC}, \textbf{Ross-Ibarra J},  Flint-Garcia S,  Chen X,  Springer NM, Doebley JF (2019). Hybrid decay: a transgenerational epigenetic decline in vigor and viability triggered in backcross populations of teosinte with maize. \textsc{Genetics} 213: 143-160
%CITES:10736570628617082030

\item \textbf{O'Brien AM}$^\S$, Sawers RJH, Strauss SY, \textbf{Ross-Ibarra J}$^\S$ (2019). Adaptive phenotypic divergence in an annual grass differs across biotic contexts. \textsc{Evolution} 73: 2230-2246
%CITES:7213583629655453228,1411114462101707243

\item  Gonzalez-Segovia E,  P\'erez-Limon S,  C\'intora-Mart\'inez C,  Guerrero-Zavala A,  Jansen G,  Hufford MB, \textbf{Ross-Ibarra J}, Sawers RJH (2019). Characterization of introgression from the teosinte \textit{Zea mays} ssp. \textit{mexicana} to Mexican highland maize. \textsc{PeerJ} 7: e6815. %doi: 10.7287/peerj.preprints.27489v1
%CITES:8050740196543639739

\item \textbf{Gates DJ}$^\S$, Runcie D, Janzen GM, Romero Navarro A,  Willcox M,  Sonder K, Snodgrass SJ, Rodr\'{i}guez-Zapata F,  Sawers RJH,  Rell\'{a}n-\'{A}lvarez R, Buckler ES, Hearne S, Hufford MB, \textbf{Ross-Ibarra J}$^\S$ (2019). Single-gene resolution of locally adaptive genetic variation in Mexican maize. \textsc{bioRxiv} 706739; doi: 10.1101/706739
%CITES:14594985058292814073

\item \textbf{Josephs EM}$^\S$, Berg JJ, \textbf{Ross-Ibarra J}, Coop G (2019) Detecting adaptive differentiation in structured populations with genomic data and common gardens. \textsc{Genetics} 211: 989-1004.
%CITES:4120965274846987962

\item \textbf{Stetter MG}$^\S$, Thornton K, \textbf{Ross-Ibarra J}$^\S$ (2018) Genetic architecture and selective sweeps after polygenic adaptation to distant trait optima. \textsc{PLoS Genetics} 14(11): e1007794. %doi: 10.1101/313247
%CITES:3273553693892067877

\item \textbf{O'Brien A}$^\S$, Sawers R, \textbf{Ross-Ibarra J}, Strauss  SY$^\S$ (2018) Evolutionary responses to conditionality in species interactions across environmental gradients. \textsc{American Naturalist} 192(6): 715-730.
%CITES:3273742166310424316,14964455021812411272,6253902269778730814,1188360045255703991

\item \textbf{Stitzer MC}$^\S$, \textbf{Ross-Ibarra J} (2018) Maize domestication and gene interaction. \textsc{New Phytologist} 220:395-408
%CITES:9512221546668576266

\item Manchanda N, Snodgrass SJ, \textbf{Ross-Ibarra J}, Hufford MB (2018) Evolution and adaptation in the maize genome. pages 319-332 \textit{In} \textsc{The Zea mays Genome}, Bennetzen, Flint-Garcia, Hirsch, Tuberosa (Eds.), Springer Nature Publishing 
%CITES:17712083587418275641

\item \textbf{Lorant A}, \textbf{Ross-Ibarra J}, Maud Tenaillon (2018) Genomics of long- and short- term adaptation in maize and teosinte. Pages 289-311 \textit{In} \textsc{Statistical Population Genomics},  Dutheil (Ed.), Springer Nature Publishing 
%CITES:16941674438635770595

\item Dawe RK, Lowry EG, Gent J, \textbf{Stitzer MC}, Higgins DM, \textbf{Ross-Ibarra J}, Wallace JG, Kanizay L, Alabady M, Wang N, Gao Z, Birchler J, Harkess AE, Hodges AL, Hiatt EN (2018) A novel maize kinesin causes neocentromere activity and meiotic drive, altering inheritance patterns across the genome. \textsc{Cell} 173: 839-850.
%CITES:7814826500218882810

\item  Aburto-Oropeza O, Johnson A, Agha M, Allen E,  Allen M,  Gonz\'{a}lez JA, Arenas-Moreno DM, Beas R, Butterfield H, Caetano G, Caselle J, Caste\~{n}ada Gayt\'{a}n G, Castorani MCN, Anh Cat L, Cavanaugh K, Chambers JQ, Cooper RD, Arafeh-Dalmau N, Dawson T, Diaz de la Vega A, DiMento JFC, Dom\'{i}nguez S, Edwards M, Ennen J, Estrada-Medina H, Fierro N, Gadsden H, Galina-Tessaro P, Gibbons P, Goode EV, Gorris ME, Harmon T, Hecht SB, Heredia Fragoso MA, Hern\'{a}ndez-Solano A, Hern\'{a}ndez-Cort\'{e}s D, Hern\'{a}ndez-Carmona G, Hillard S, Huey RB, Hufford MB, P\`{a}ramo Figueroa VH, Jenerette D, Jim\'{e}nez-Osornio J, L\'{o}pez-Nava KJ, Lara R, Leslie H, Lopez-Feldman A, Luja V, Mart\'{i}nez-M\'{e}ndez N, Mautz W, Medellin-Azuara J, Mel\'{e}ndez-Torres C, de la Cruz FRM,  Micheli F, Miles D, Montagner G, Monta\~{n}o-Moctezuma G, M\"{u}ller J, Oliva P, Ortinez A, Ortiz Partida JP, Palleiro-Nayar J, Parnell PE, Raimondi P, Ramirez A, Randerson JT, Reed DC, Riquelme M, Torres TR, Rosen PC, {\bf Ross-Ibarra J}, Sanchez-Cordero V, Sandoval-Solis S, Santos J, Sawers R, Sinervo B, Sites J,  Sosa-Nishizaki O, Stanton T, Stapp J, Stewart J, Torre J,  Torres-Moye G, Treseder KK, Valdez-Villavicencio JH, Jim\'{e}nez FIV, Vaughn M, Welton L, Westphal MF, Woolrich-Pi\~{n}a G,  Yunez-Naude A, Zertuche-Gonz\'{a}lez JA, Taylor JE (2018)
Harnessing Cross-border Resources to Confront Climate Change.  \textsc{Environmental Science and Policy}  87: 128-132.
%CITES:11343709371087280313

\item \textbf{Bilinski P}$^\S$, Albert P, Berg JJ, Birchler JA, Grote M, \textbf{Lorant A}, \textbf{Quezada J}$^\ddagger$, Swarts, K, \textbf{Yang J}, \textbf{Ross-Ibarra J}$^\S$ (2018) Parallel altitudinal clines reveal adaptive evolution of genome size in \textit{Zea mays}. \textsc{PLoS Genetics} 14: e1007162
%CITES:6345641206578854017,429102382613440114,16686636666309032266

\item {\bf Mei W}, {\bf Stetter MG}, {\bf Gates DJ}, {\bf Stitzer MC}, {\bf Ross-Ibarra J}$^\S$ (2018) Adaptation in plant genomes: bigger is different.  \textsc{American Journal of Botany} 105: 16-19
%CITES:9469285840365544650

\item Bukowski R, Guo X, Lu Y, Zou C, He B, Rong Z, Wang B,  Xu D,  Yang B, Xie C,  Fan L, Gao S, Xu X, Zhang G, Li Y, Jiao Y, Doebley J, {\bf Ross-Ibarra J}, {\bf Lorant A}, {\bf Buffalo V}, Romay MC, Buckler ES, Ware D, Lai J, Sun Q , Xu Y (2017) Construction of the third generation \emph{Zea mays} haplotype map. \textsc{GigaScience} gix134
%CITES:9019483855092094967
%\href{http://biorxiv.org/content/early/2015/09/16/026963}{}

\item Wang L, \textbf{Beissinger TM}, \textbf{Lorant A}, \textbf{Ross-Ibarra C}, \textbf{Ross-Ibarra J}$^\S$, Hufford MB$^\S$ (2017) The interplay of demography and selection during maize domestication and diffusion. \textsc{Genome Biology} 18:215
%CITES:11040677895022456988

\item {\bf Yang J}$^{*\S}$, {\bf Mezmouk S}$^*$, Baumgarten A, Buckler ES, Guill KE, McMullen MD, Mumm RH, {\bf Ross-Ibarra J}$^\S$ (2017) Incomplete dominance of deleterious alleles contribute substantially to trait variation and heterosis in maize. \textsc{PLoS Genetics} 13:e1007019
%CITES:10986799122751558384

\item \textbf{Lorant A}, Pedersen S, Holst I, Hufford MB, Winter K, Piperno D, \textbf{Ross-Ibarra J}$^\S$ (2017) The potential role of genetic assimilation during maize domestication. \textsc{PLoS ONE} 12:e0184202
%CITES:7419221198329577195

\item Aguilar-Rangel MR, Ch\`{a}vez Montes RA,  Gonzalez-Segovia E, \textbf{Ross-Ibarra J}, Simpson JK, Sawers RJH (2017) Allele specific expression analysis identifies regulatory variation associated with stress-related genes in the Mexican highland maize landrace Palomero Toluque\~{n}o. \textsc{PeerJ} 5:e3737
%CITES:11847404813452014186

\item   {\bf Stetter MG}$^\S$, {\bf Gates DJ}, {\bf Mei W}, {\bf Ross-Ibarra J}$^\S$ (2017) How to make a domesticate. \textsc{Current Biology} 27:R896-R900
%CITES:8720581075310602102

\item Swarts K, Gutaker RM,   Schuenemann V,  Benz B,  Blake M, Bukowski R, Holland J, Kruse-Peeples M, Lepak N, Matson RG, Prim L, Romay C, {\bf Ross-Ibarra J}, Sanchez J, Schmidt C,  Sofro E,  Krause J,  Weigel D,  Buckler ES,  Burbano HA (2017) Genomic estimation of complex traits reveals ancient maize adaptation to temperate North America. \textsc{Science} 357:512-515
%CITES:10777136530412179569

\item {\bf Bilinski P}$^\S$, Han Y, {\bf Hufford MB}, {\bf Lorant A}, Zhang P, Jiang J, {\bf Ross-Ibarra J}$^\S$ (2017) Genomic abundance is not predictive of tandem repeat localization in grass genomes. \textsc{PLoS ONE} 12:e0177896
%CITES:13486348739945366922,2898437834389851132

\item  Jiao Y, Peluso P,  Shi J,  Liang T, {\bf Stitzer MC}, Wang B,  Campbell M, Stein JC,  Wei X,  Chin C-S,  Guill K,  Regulski M,  Kumari S,  Olson A,  Gent J, Schneider KL,  Wolfgruber TK, May MR, Springer N,  Antoniou E,  McCombie R, Presting GG,  McMullen M, {\bf Ross-Ibarra J}, Dawe RK,  Hastie A, Rank DR, Ware D (2017) Improved maize reference genome with single-molecule technologies. \textsc{Nature}  546:524-527
%CITES:17766642909868058130,10921800585070774953

\item  {\bf Renny-Byfield S}$^\S$, Rodgers-Melnick E, {\bf Ross-Ibarra J}$^\S$ (2017) Gene fractionation and function in the ancient subgenomes of maize. \textsc{Mol. Biol. Evol.} 34:1825-1832
%CITES:7118994421153889601

\item {\bf Velasco D}, Aradhya M, and  and {\bf Ross-Ibarra J}$^\S$ (2016) Evolutionary genomics of peach and almond domestication. \textsc{G3} 6:3985-3993
%CITES:4675131869514095820

\item Ramos-Madrigal J, Smith BD, Moreno-Mayar JV, Gopalakrishnan S, {\bf Ross-Ibarra J}, Gilbert MTP, Wales N (2016) Genome sequence of a 5310-year-old maize cob provides insights into the early stages of maize domestication. \textsc{Current Biology} 26:3195-3201
%CITES:4675131869514095820

\item {\bf Durvasula A}$^\ddagger$$^*$,  Hoffman PJ$^*$, {\bf Kent TV}$^\ddagger$, Liu C, Kono TJY, Morrell PL$^\S$, {\bf Ross-Ibarra J}$^\S$ (2016) ANGSD-wrapper. \textsc{Molecular Ecology Resources} 16:1449-1454
%CITES:14658936176419871996,405592559232898639

\item {\bf Beissinger TM}$^\S$, Wang L, {\bf Crosby K}, {\bf Durvasula A}$^\ddagger$, Hufford MB, {\bf Ross-Ibarra J}$^\S$ (2016)  Recent demography drives changes in linked selection across the maize genome. \textsc{Nature Plants} 2:16084
%\href{http://biorxiv.org/content/early/2015/11/13/031666}{Recent demography drives changes in linked selection across the maize genome}
%CITES:14270822170481133370

\item Wolfgruber TK, Nakashima MM, Schneider KL, Sharma A, Xie Z, Albert PS, Xu R, {\bf Bilinski P},  Dawe RK, {\bf Ross-Ibarra J}, Birchler JA, Presting G (2016) High quality maize centromere 10 sequence reveals evidence of frequent recombination events. \textsc{Frontiers In Plant Science} 7
%CITES:4675131869514095820

\item Orozco-Ram\`{i}rez Q, Santacruz-Varela A, {\bf Ross-Ibarra J}, Brush B (2016) Maize diversity associated with social origin and environmental variation in southern Mexico. \textsc{Heredity} 116:477-484.
%\emph{In Press -- \href{https://peerj.com/preprints/1192/}{preprint}}
%CITES:13046200421462083749,13021360675733443150


\item Gerke JP$^\S$, Edwards JW, Guill KE, {\bf Ross-Ibarra J}$^\S$, McMullen MD.  The genomic impacts of drift and selection for hybrid performance in maize (2015). \textsc{Genetics}  201: 1201-1211
%Preprint: \url{http://arxiv.org/abs/1307.7313}
%CITES:18298112768791508981


\item Sosso D, Luo D, Li Q-B, Sassse J, {\bf Yang J}, Gendrot G, Suzuki M, Koch KE, McCarty DR, Chourey PS, Rogoswky PM, {\bf Ross-Ibarra J}, Yang B, Frommer WB (2015) Seed filling in domesticated maize and rice depends on SWEET-mediated hexose transport. \textsc{Nature Genetics} 47:1489-1493
%CITES:14940779386657346302


\item {\bf Takuno S}, Ralph P, Swarts K, Elshire RJ, Glaubitz JC, Buckler ES, {\bf Hufford MB}, {\bf Ross-Ibarra J}$^\S$ (2015) Independent molecular basis of convergent highland adaptation in maize. \textsc{Genetics} 200:1297-1312
%CITES:2741664858361885827


\item {\bf Vann LE}, {\bf Kono T}, {\bf Pyh\"aj\"arvi T}, {\bf Hufford MB}$^\S$, {\bf Ross-Ibarra J}$^\S$ (2015) Natural variation in teosinte at the domestication locus teosinte branched1 (tb1). \textsc{PeerJ} 3:e900
%CITES:4502892611533604349


\item Hake S, {\bf Ross-Ibarra J} (2015) Genetic, evolutionary and plant breeding insights from the domestication of maize. \textsc{eLife}  2015;4:e05861
%CITES:15579853227920259642


\item Fonseca RR, Smith B, Wales N, Cappellini E, Skoglund P, Fumagalli M, Samaniego JA, Caroe C, Avila-Arcos MC, Hufnagel D, Korneliussen TS, Vieira FG, Jakobsson M, Arriaza B, Willerslev E, Nielsen R, Hufford MB, Albrechtsen A,  {\bf Ross-Ibarra J}, Gilbert MT (2015) The origin and evolution of maize in the American Southwest. \textsc{Nature Plants} 1:14003
%CITES:14394472478509816248,10810382578079898537,10712802329193176067

\item Dyer GA, L\'opez-Feldman A, Y\'unez-Naude A, Taylor JE, {\bf Ross-Ibarra J} (2015) Reply to Brush \emph{et al.}: A wake up call for crop conservation science. PNAS 112 (1), E2-E2 (letter).
%CITES:10306546315448360067


\item Makarevitch I, Waters M, West P, {\bf Stitzer M}, {\bf Ross-Ibarra, J}, Springer NM (2015) Mobile elements contribute to activation of genes in response to abiotic stress. \textsc{PLoS Genetics} 11 (1): e1004915. %Preprint: \url{http://biorxiv.org/content/early/2014/08/15/008052}
%CITES:726174711625580935


\item Tiffin P, {\bf Ross-Ibarra J} (2014) Advances and limits of using population genetics to understand local adaptation. \textsc{Trends in Ecology and Evolution} 29:673-680 %Preprint: \url{https://peerj.com/preprints/488/}
%CITES:2471984348452499818


\item {\bf Bilinski P}, {\bf Distor KD}, {\bf Gutierez-Lopez J}, {\bf Mendoza Mendoza G}, Shi J, Dawe RK,  {\bf Ross-Ibarra J}$^\S$ (2014) Diversity and evolution of centromere repeats in the maize genome. \textsc{Chromosoma} 0009-5915
%CITES:7474610793083089141

\item {\bf Mezmouk S}, {\bf Ross-Ibarra J}$^\S$ (2014) The pattern and distribution of deleterious mutations in maize. \textsc{G3} 4:163-171
%CITES:18217003468169681800
%Preprint: http://arxiv.org/abs/1308.0380

\item Waters AJ, {\bf Bilinski P}, Eichten SR, Vaughn MW, {\bf Ross-Ibarra J}, Gehring M, Springer NM (2013) Comprehensive analysis of imprinted genes in maize reveals allelic variation for imprinting and limited conservation with other species. \textsc{PNAS} 110:19639-19644
%Preprint: http://arxiv.org/abs/1307.7678
%CITES:16494053027693693141,2019155999972302238


\item {\bf Pyh\"aj\"arvi T}, {\bf Hufford MB}, {\bf Mezmouk S}, {\bf Ross-Ibarra J}$^\S$ (2013) Complex patterns of local adaptation in teosinte. \textsc{Genome Biology and Evolution} 5: 1594-1609 %$^\dagger$
%CITES:4348910575877017766
%Preprint: \emph{http://arxiv.org/abs/1208.0634}


\item Wills DM, Whipple C, {\bf Takuno S}, Kursel LE, Shannon LM, {\bf Ross-Ibarra J}, Doebley JF (2013) From many, one: genetic control of prolificacy during maize domestication. \textsc{PLoS Genetics} 9(6): e1003604. %Preprint: \emph{http://arxiv.org/abs/1303.0882}
%CITES:6107325531042162678

\item McCouch S, Baute GJ, Bradeen J, Bramel P, Bretting PK, Buckler E, Burke JM, Charest D, Cloutier S, Cole G, Dempewolf H, Dingkuhn M, Feuillet C, Gepts, P, Grattapaglia D, Guarino L, Jackson S, Knapp S, Langridge P, Lawton-Rauh A, Lijua Q, Lusty C, Michael T, Myles S, Naito K, Nelson RL, Pontarollo R, Richards CM, Rieseberg L, {\bf Ross-Ibarra J}, Rounsley S, Hamilton RS, Schurr U, Stein N, Tomooka N, van der Knaap E, van Tassel D, Toll J, Valls J, Varshney RK, Ward J, Waugh R, Wenzl P, Zamir. (2013) Agriculture: Feeding the future. \textsc{Nature} 499:23-24
%CITES:17831095819213676676,4520310780634611218,7413676098539789267

\item {\bf Hufford MB}, Lubinsky P, {\bf Pyh\"aj\"arvi T}, {\bf Devengenzo MT}$^\ddagger$, Ellstrand NC, {\bf Ross-Ibarra J}$^\S$ (2013) The genomic signature of crop-wild introgression in maize. \textsc{PLoS Genetics} 9(5): e1003477. %Preprint: \emph{http://arxiv.org/abs/1208.3894}
%CITES:16496318242013452561,5236304695133591580


\item {\bf Provance MC}$^\S$, Garcia Ruiz I, {\bf Thommes C}$^\ddagger$, {\bf Ross-Ibarra J} (2013) Population genetics and ethnobotany of cultivated \emph{Diospyros riojae} G\'omez Pompa (Ebenaceae), an endangered fruit crop from Mexico. \textsc{Genetic Resources and Crop Evolution} 60: 2171-2182.
%Preprint: \emph{http://dx.doi.org/10.6084/m9.figshare.105841}
%CITES:15526564212076251883


\item Melters DP$^*$, Bradnam KR$^*$, Young HA, Telis N, May MR, Ruby JG, Sebra R, Peluso P, Eid J, Rank D, Fernando Garcia J, DeRisi J, Smith T, Tobias C, {\bf Ross-Ibarra J}$^\S$, Korf IF$^\S$, Chan SW-L. (2013) Patterns of centromere tandem repeat evolution in 282 animal and plant genomes. \textsc{Genome Biology} 14:R10
%Preprint: \emph{http://arxiv.org/abs/1209.4967}
%CITES:4643344870051815303


\item Kanizay LB, {\bf Pyh\"aj\"arvi T}, Lowry E, {\bf Hufford MB}, Peterson DG, {\bf Ross-Ibarra J}, Dawe RK (2013) Diversity and abundance of the Abnormal chromosome 10 meiotic drive complex in \emph{Zea mays}. \textsc{Heredity} 110: 570-577.
%CITES:9054275167516914311


\item {\bf Hufford MB}, {\bf Bilinski P}, {\bf Pyh\"aj\"arvi T}, {\bf Ross-Ibarra J}$^\S$ (2012) Teosinte as a model system for population and ecological genomics. \textsc{Trends in Genetics} 12:606-615 %$^\dagger$
%CITES:17241763016461383750


\item Mu\~{n}oz Diez C, Vitte C, {\bf Ross-Ibarra J}, Gaut BS, Tenaillon MI (2012) Using nextgen sequencing to investigate genome size variation and transposable element content. \emph{In} Grandbastien M-A, Casacuberta JM, editors. \textsc{Topics in Current Genetics} v24: Plant Transposable Elements - Impact on Genome Structure \& Function. pp. 41-58
%CITES:5019892223730894925


\item  {\bf van Heerwaarden J}$^\S$, {\bf Hufford MB}, {\bf Ross-Ibarra J}$^\S$ (2012) Historical genomics of North American maize. \textsc{PNAS} 109: 12420-12425
%CITES:3745727334869718431


\item Swanson-Wagner R, Briskine R, Schaefer R, {\bf Hufford MB}, {\bf Ross-Ibarra J}, Myers CL, Tiffin P, Springer NM.  Reshaping of the maize transcriptome by domestication. (2012). \textsc{PNAS}  109: 11878-11883
%CITES:935006617287790715


\item {\bf Hufford MB}$^*$, Xun X$^*$, {\bf van Heerwaarden J}$^*$, {\bf Pyh\"aj\"arvi T}$^*$, Chia J-M, Cartwright RA, Elshire RJ, Glaubitz JC, Guill KE, Kaeppler S, Lai J, Morrell PL, Shannon LM, Song C, Spinger NM, Swanson-Wagner RA, Tiffin P, Wang J, Zhang G, Doebley J, McMullen MD, Ware D, Buckler ES$^\S$, Yang S$^\S$, {\bf Ross-Ibarra J}$^\S$ (2012) Comparative population genomics of maize domestication and improvement. \textsc{Nature Genetics} 44:808-811 %$^\dagger$
%CITES:2254239507517002236


\item  Chia J-M$^*$, Song C$^*$, Bradbury P, Costich D, de Leon N, Doebley JC, Elshire RJ, Gaut BS, Geller L, Glaubitz JC, Gore M, Guill KE, Holland J,  {\bf Hufford MB}, Lai J, Li M, Liu X, Lu Y, McCombie R, Nelson R, Poland J, Prasanna BM,  {\bf Pyh\"aj\"arvi T}, Rong T, Sekhon RS,  Sun Q, Tenaillon M, Tian F, Wang J, Xu X, Zhang Z, Kaeppler S, {\bf Ross-Ibarra J}, McMullen M, Buckler ES, Zhang G, Xu Y, Ware, D (2012) Maize HapMap2 identifies extant variation from a genome in flux. \textsc{Nature Genetics} 44:803-807 %$^\dagger$
%CITES:2228837645322681220


\item Fang Z, {\bf Pyh\"aj\"arvi T}, Weber AL, Dawe RK, Glaubitz JC, S\'{a}nchez Gonz\'{a}lez J, {\bf Ross-Ibarra C}, Doebley J, Morrell PL$^\S$, {\bf Ross-Ibarra J}$^\S$  (2012) Megabase-scale inversion polymorphism in the wild ancestor of maize. \textsc{Genetics} 191:883-894
%CITES:6473644606884467791


\item Cook JP, McMullen MD, Holland JB, Tian F, Bradbury P, {\bf Ross-Ibarra J}, Buckler ES, Flint-Garcia SA (2012) Genetic architecture of maize kernel composition in the Nested Association Mapping and Inbred Association panels. \textsc{Plant Physiology} 158: 824-834
%CITES:11097878024256170139

\item Morrell PL, Buckler ES, {\bf Ross-Ibarra J}$^\S$ (2012) Crop genomics: advances and applications.  \textsc{Nature Reviews Genetics} 13:85-96 %$^\dagger$
%CITES:12664715338521683012


\item Studer A, Zhao Q, {\bf Ross-Ibarra J}, Doebley J (2011) Identification of a functional transposon insertion in the maize domestication gene \emph{tb1}.  \textsc{Nature Genetics} 43:1160-1163.
%CITES:11888569066379603503,5440450712221885825


\item {\bf van Heerwaarden J}$^\S$, Doebley J, Briggs WH, Glaubitz JC, Goodman MM, S\'{a}nchez Gonz\'{a}lez JJ, {\bf Ross-Ibarra J}$^\S$ (2011) Genetic signals of origin, spread and introgression in a large sample of maize landraces. PNAS 108: 1088-1092
%CITES:11944729109494726624


\item {\bf Hufford MB}$^\S$, Gepts P, {\bf Ross-Ibarra J} (2011) Influence of cryptic population structure on observed mating patterns in the wild progenitor of maize (\emph{Zea mays} ssp. \emph{parviglumis}).  \textsc{Molecular Ecology} 20: 46-55
%CITES:13621041706202411296


\item Tenaillon MI, {\bf Hufford MB}, Gaut BS, {\bf Ross-Ibarra J}$^\S$ (2011)  Genome size and TE content as determined by high-throughput sequencing in maize and \emph{Zea luxurians}.  \textsc{Genome Biology and Evolution } 3: 219-229
%CITES:8095218463542481067


\item Eckert AJ, {\bf van Heerwaarden J}, Wegrzyn JL, Nelson CD, {\bf Ross-Ibarra J}, Gonz\'{a}lez-Mart\'{i}nez SC, and Neale DB (2010) Patterns of population structure and environmental associations to aridity across the range of loblolly pine (\emph{Pinus taeda} L, Pinaceae).  \textsc{Genetics} 185: 969-982
%CITES:8457237997922496538,3590125064334610687


\item Fuchs EJ, {\bf Ross-Ibarra J}$^\S$, Barrantes G (2010) Reproductive biology of \emph{Macleania rupestris}: a pollen-limited Neotropical cloud-forest species in Costa Rica.  \textsc{Journal of Tropical Ecology} 26: 351-354
%CITES:14583174141795727444


\item Whitney KD, Baack EJ, Hamrick JL, Godt MJW, Barringer BC, Bennett MD, Eckert CG, Goodwillie C, Kalisz S, Leitch I, {\bf Ross-Ibarra J} (2010) A role for nonadaptive processes in plant genome size evolution? . \textsc{Evolution} 64: 2097-2109
%CITES:17559374760333968488


\item {\bf van Heerwaarden J}, {\bf Ross-Ibarra J}$^\S$, Doebley J, Glaubitz JC, S\'{a}nchez Gonz\'{a}lez J, Gaut BS, Eguiarte LE (2010) Fine scale genetic structure in the wild ancestor of maize (\emph{Zea mays} ssp. \emph{parviglumis}).  \textsc{Molecular Ecology} 19: 1162-1173
%CITES:9340353919503578071


\item Shi J, Wolf S, Burke J, Presting G, {\bf Ross-Ibarra J}, Dawe RK (2010) High frequency gene conversion in centromere cores.  \textsc{PLoS Biology} 8: e1000327
%CITES:15554747939087267209


\item Hollister JD, {\bf Ross-Ibarra J}, Gaut BS (2010) Indel-associated mutation rate varies with mating system in flowering plants.  \textsc{Molecular Biology and Evolution} 27: 409-416.
%CITES:1965523728393558029

\item {\bf van Heerwaarden J}, van Eeuwijk FA, {\bf Ross-Ibarra J} (2010) Genetic diversity in a crop metapopulation.  \textsc{Heredity} 104: 28-39
%CITES:15394221802384617228


\item Gore MA$^*$, Chia JM$^*$, Elshire RJ, Sun Q, Ersoz ES, Hurwitz BL, Peiffer JA, McMullen MD, Grills GS, {\bf Ross-Ibarra J}, Ware D, Buckler ES (2009) A first-generation haplotype map of maize.  \textsc{Science 326}: 1115-1117.
%CITES:1250430020833640405


\item {\bf May MR}$^\ddagger$, {\bf Provance MC}, Sanders AC, Ellstrand NC, {\bf Ross-Ibarra J}$^\S$ (2009) A pleistocene clone of Palmer's Oak persisting in Southern California.  \textsc{PLoS ONE} 4: e8346.
%CITES:15690173176823572703,5159519491829986435


\item Zhang LB, Zhu Q, Wu ZQ, {\bf Ross-Ibarra J}, Gaut BS, Ge S, Sang T (2009) Selection on grain shattering genes and rates of rice domestication.  \textsc{New Phytologist} 184: 708-720.
%CITES:853064656161512620

\item {\bf Ross-Ibarra J}, Tenaillon M, Gaut BS (2009) Historical divergence and gene flow in the genus Zea.  \textsc{Genetics} 181: 1399-1413.
%CITES:12731906589694221305


\item {\bf Ross-Ibarra J}$^*$, Wright SI$^*$, Foxe JP, Kawabe A, DeRose-Wilson L, Gos G, Charlesworth D, Gaut BS (2008) Patterns of polymorphism and demographic history in natural populations of \emph{Arabidopsis lyrata}.  \textsc{PLoS ONE} 3: e2411.
%CITES:12619209345184003027


\item Lockton S, {\bf Ross-Ibarra J}, Gaut BS (2008) Demography and weak selection drive patterns of transposable element diversity in natural populations of \emph{Arabidopsis lyrata}. PNAS 105: 13965-13970.
%CITES:15151328051935459123


\item {\bf Ross-Ibarra J}$^\S$, Gaut BS (2008) Multiple domestications do not appear monophyletic. PNAS 105: E105 (letter).
%CITES:14659039718259909589


\item Gaut BS, {\bf Ross-Ibarra J} (2008) Selection on major components of angiosperm genomes.  \textsc{Science} 320: 484-486.
%CITES:10471445648121255184


\item {\bf Ross-Ibarra J}, Morrell PL, Gaut BS (2007) Plant domestication, a unique opportunity to identify the genetic basis of adaptation. PNAS 104 Suppl 1: 8641-8648.
%CITES:4061357872113450280


\item {\bf Ross-Ibarra J}$^\S$ (2007) Genome size and recombination in angiosperms: a second look.  \textsc{Journal of Evolutionary Biology} 20: 800-806.
%CITES:684436662966421061


\item Wares JP, Barber PH, {\bf Ross-Ibarra J}, Sotka EE, Toonen RJ (2006) Mitochondrial DNA and population size.  \textsc{Science} 314: 1388-90 (letter).
%CITES:17853791691398286654


\item {\bf Ross-Ibarra J}$^\S$ (2005) QTL and the study of plant domestication.  \textsc{Genetica} 123: 197-204.
%CITES:7426372046771115962


\item {\bf Ross-Ibarra J}$^\S$ (2004) The evolution of recombination under domestication: a test of two hypotheses.  \textsc{American Naturalist} 163: 105-112.
%CITES:3095269849835314095


\item {\bf Ross-Ibarra J} (2003) Origen y domesticaci\'on de la chaya (Cnidoscolus aconitifolius Mill IM Johnst): La espinaca Maya (\emph{Cnidoscolus aconitifolius} Mill I. M. Johnst): Mayan spinach.  \textsc{Mexican Studies} 19: 287-302.
%CITES:4811395532917369896

\item {\bf Ross-Ibarra J}$^\S$, Molina-Cruz A (2002) The ethnobotany of Chaya (\emph{Cnidoscolus aconitifolius} ssp. \emph{aconitifolius} Breckon): A nutritious Maya vegetable. \textsc{Economic Botany} 56: 350-365.
%CITES:5302092154685965393

\item  Neel MC, {\bf Ross-Ibarra J}, Ellstrand NC (2001) Implications of mating patterns for conservation of the endangered plant \emph{Eriogonum ovalifolium} var. \emph{vineum}.  \textsc{American Journal of Botany} 88: 1214-1222.
%CITES:13724877679765824056
\end{etaremune}

%OTHER PUBS
%\subsection*{Selected other publications}
%\begin{itemize}
%\item Close, T, R Last and 22 co-authors (2011) Phenomics: Genotype to Phenotype. White paper to the US National Science Foundation and US Department of Agriculture.

%\item Ross-Ibarra, J, PL Morrell and BS Gaut (2007) Plant Domestication, a Unique Opportunity to Identify the Genetic Basis of Adaptation. In: In the Light of Evolution Editors: FJ Ayala and J Avise The National Academies Press Washington, DC
%\item Ross-Ibarra, J (2005) QTL Mapping and the study of plant domestication. In: Genetics of Adaptation Springer, Dordrecht
%\end{itemize}

%\newpage
%
%
%\section*{References}
%\begin{itemize}
%\item {\bf Brandon S. Gaut}
%\begin{itemize}
%\item Professor
%\item Dept. of Ecology and Evolutionary Biology
%\item University of California
%\item Irvine, CA. 92697
%\item tel:  (949) 824-2564
%\item email: bgaut@uci.edu
%\end{itemize}
%
%\item {\bf John Doebley}
%\begin{itemize}
%\item Professor
%\item Genetics Dept.
%\item University of Wisconsin
%\item 425-G Henry Mall
%\item Madison, WI 53706
%\item tel: (608) 265-5803
%\item email:  jdoebley@wisc.edu
%\end{itemize}
%
%\item {\bf Edward Buckler}
%\begin{itemize}
%\item USDA-ARS Research Geneticist \& Adjunct Professor
%\item Institute for Genomic Diversity
%\item Cornell University
%\item 159 Biotechnology Bldg
%\item Ithaca, NY 14853-2703
%\item tel: (607) 255-4520
%\item email: esb33@cornell.edu
%\end{itemize}
%\end{itemize}

% Footer
%\begin{center}
%  \begin{footnotesize}
%    Last updated: \today \\
%    \href{\footerlink}{\texttt{\footerlink}}
%  \end{footnotesize}
%\end{center}

\end{document}
